From 86413e5f01339ce54bcece2d1d8b1b88d8823c1e Mon Sep 17 00:00:00 2001
From: Alexey Tikhonov <atikhono@redhat.com>
Date: Tue, 26 Oct 2021 16:02:43 +0200
Subject: [PATCH 14/17] SUDO: decrease log level in case object wasn't found

It is expected sudo responder can be requested to lookup unknown entry.
One of typical examples is lookup for a local user.

Resolves: https://github.com/SSSD/sssd/issues/5839

Reviewed-by: Pawel Polawski <ppolawsk@redhat.com>
---
 src/responder/sudo/sudosrv_cmd.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/responder/sudo/sudosrv_cmd.c b/src/responder/sudo/sudosrv_cmd.c
index 3bed22b6fc8b476686269d68e49def6a5af9383b..63b548fe8d5c76aa2fb6eec7f1b174fa7f47f90b 100644
--- a/src/responder/sudo/sudosrv_cmd.c
+++ b/src/responder/sudo/sudosrv_cmd.c
@@ -261,7 +261,8 @@ static void sudosrv_cmd_done(struct tevent_req *req)
                                  &cmd_ctx->num_rules);
     talloc_zfree(req);
     if (ret != EOK) {
-        DEBUG(SSSDBG_CRIT_FAILURE, "Unable to obtain cached rules [%d]: %s\n",
+        DEBUG((ret == ENOENT) ? SSSDBG_MINOR_FAILURE : SSSDBG_OP_FAILURE,
+              "Unable to obtain cached rules [%d]: %s\n",
               ret, sss_strerror(ret));
         goto done;
     }
-- 
2.31.1

